### PR TITLE
add: alert and button 

### PR DESCRIPTION
**Pull request by 05him**
added alert and button in the component library 

live URL:-https://05him.github.io/Dark_UI/

### CODE REVIEW

**Review by vrx29 — APPROVED** [review]
Nice UI and logo. Looks good to me

File: components/components.css [hunk 1]
```diff
@@ -18,11 +20,18 @@
   --primary-bg: rgba(48, 140, 122, 0.3);
   --secondary-color: #03071e;
   --text-color: white;
+
   --demo-bg: #d0cbcbd9;
   --badge-bg: red;
   --badge-online: rgb(3, 214, 3);
```
Comment (vrinda-mahajan) [inline components/components.css:26]: Just a suggestion, either use hex values or rgb values for defining colors.

**Review by vrinda-mahajan — APPROVED** [review]
Everything else looks good to me.

**Review by MayankKumar10 — APPROVED** [review]
Looks good to me you can merge it.

File: components/components.css (hunk 1 above)
Comment (05him) [reply components/components.css:26]: oky sure will do it

**Pull request merged**